server/app.py

```diff
@@ -177,6 +183,7 @@ async def submit_batch(batch: BatchData):
     try:
         for record in batch.data:
             await producer.send_and_wait(topic, value=record)
+            time.sleep(0.1)
```

Comment:
sorry, can we just leave a comment explaining this is a temp fix? ideally with a loud FIXME in front